funnel/templates/index.html.jinja2

```diff
@@ -371,7 +372,7 @@
               <tbody>
                 {% for project in past_projects %}
                   <tr>
-                    <td data-th="Date"><p class="mui--text-subhead zero-bottom-margin">{{ project.schedule_start_at|longdate }}</p></td>
+                    <td data-th="Date"><p class="mui--text-subhead zero-bottom-margin">{{ project.schedule_start_at_localized.strftime('%d %B %Y')}}</p></td>
```

Comment:
Avoid `strftime` as it breaks i18n. Use the appropriate filter. If we don't have one, we can add it.

Reply:
https://github.com/hasgeek/baseframe/pull/270 adds these filters